scripts/acir_migrate_syntax.py

```
@@ -0,0 +1,614 @@
+- Primitive selection (placeholders must be replaced).
+- Circuit parameter type inference (defaults to `real`).
+- Complex sizing/parameter expressions.
+"""
+
+from __future__ import annotations
+
+import argparse
+import re
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Iterable
```

Ruff (0.14.14)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)